020/main.go

```diff
@@ -0,0 +1,24 @@
+// Daily Joel #20
+// Methods.
+package main
+
+import "fmt"
+
+type Person struct {
+	name string
+}
+
+func (p *Person) setName(name string) {
+	p.name = name
+}
+
+func (p Person) setNameWithoutEffect(name string) {
+	p.name = name
+}
```

Comment:
And yes, this is where the similarities with 20 are real. Bc, again, here we're calling a copy of arguments p type person and not modifying the type itself